_saas-integrations/ebay/v1/ebay-v1.md

```diff
@@ -99,6 +99,8 @@ setup-steps:
     content: |
       {% include integrations/saas/setup/historical-sync.html %}
 
+      To retreive data from the `orders` table, the historical start date must be within 90 days.
```

Comment:
```suggestion
      **Note**: To retrieve data from the `orders` table, the **Start Date** must not exceed 90 days from the current date, whether you're initially creating the integration or resetting this parameter. If this exceeds 90 days, you'll encounter [errors during Extraction](#orders).
```

This clarifies what exactly we mean when we say `90 days`. It also links right to the documentation for the affected table, where you'll include the error message the user would receive if they didn't heed the warning. :)